ext/curl/tests/curl_persistent_share_003.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+Curl persistent share handle test with invalid option
+--EXTENSIONS--
+curl
+--FILE--
+<?php
+
+try {
+    $sh = curl_share_init_persistent([CURL_LOCK_DATA_DNS, CURL_LOCK_DATA_CONNECT, 30]);
+} catch (\ValueError $e) {
+	echo $e->getMessage() . PHP_EOL;
```

Comment:
```suggestion
    echo $e->getMessage() . PHP_EOL;
```

and below

Reply:
ty, I had copied this from another curl test and didn't realize I was mixing tabs and spaces since it looks aligned in my editor.